dist/stat_data.py

```diff
@@ -697,6 +699,8 @@ def __init__(self, name, desc, flags=''):
     CacheStat('cache_read_deleted', 'pages read into cache after truncate'),
     CacheStat('cache_read_deleted_prepared', 'pages read into cache after truncate in prepare state'),
     CacheStat('cache_read_lookaside', 'pages read into cache requiring cache overflow entries'),
+    CacheStat('cache_read_lookaside_skew_old', 'pages read into cache requiring cache overflow entries because of page is skewed with old entries'),
```

Comment:
Same comment as before regarding the descriptions.

Reply:
corrected.